src/type.ts

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable */
+import { Database } from 'sql.js'
+
+export interface IPastelDBState {
+  pastelDB: Database
+  isCreated: boolean
+}
+
+export interface IStatisticInfo {
+  hashrate: string
+  miner_distribution: string
+  difficulty: string
+}
+
+export interface INetworkInfo {
+  version: number
+  subversion: string
+  protocolversion: number
+  localservices: string
+  timeoffset: number
+  connections: number
+  networks: any
+  relayfee: number
+  localaddresses: any
+  warnings: string
+  create_timestamp: string
+}
+
+export interface INetTotals {
+  totalbytesrecv: number
+  totalbytessent: number
+  timemillis: number
+}
+
+export interface IMempoolInfo {
+  size: number
+  bytes: number
+  usage: number
+}
+
+export interface IRawMempoolInfo {
+  transactionid: string
+  size: number
+  fee: number
+  time: number
+  height: number
+  startingpriority: number
+  currentpriority: number
+  depends: any
```

Comment:
We should not have any `any` types in our code :) It defeats the purpose of TypeScript. Please change it to the actual type everywhere in this PR